src/main/java/seedu/toluist/model/Task.java

```diff
@@ -13,8 +13,8 @@
     // List of tags is unique
     private HashSet<Tag> allTags = new HashSet<>();
     public String description;
-    private LocalDateTime endDateTime;
-    private LocalDateTime startDateTime;
+    public LocalDateTime endDateTime;
+    public LocalDateTime startDateTime;
```

Comment:
alllowing these to be public means the validation constraint can be violated unknowingly when setting these values